tests/filecheck/dialects/llvm/example.mlir

```diff
@@ -98,4 +98,8 @@ builtin.module {
   %fval3 = llvm.fpext %fval : f32 to f64
 
 // CHECK-NEXT: %fval3 = llvm.fpext %fval : f32 to f64
+
+	"llvm.unreachable"() : () -> ()
```

Comment:
This seems strangely indented (would also be nice to include the custom format if you choose to include it)